packages/augur-simplified/src/modules/portfolio/portfolio-view.tsx

```diff
@@ -2,48 +2,60 @@ import React, { useState, useEffect } from 'react';
 import Styles from 'modules/portfolio/portfolio-view.styles.less';
 import { AppViewStats } from 'modules/common/labels';
 import Activity from './activity';
-import { PositionsLiquidityViewSwitcher } from 'modules/common/tables';
-import { useAppStatusStore } from 'modules/stores/app-status';
+import { PositionsLiquidityViewSwitcher } from '../common/tables';
+import { useAppStatusStore } from '../stores/app-status';
 import { PrimaryButton } from '../common/buttons';
 import { NetworkMismatchBanner } from '../common/labels';
+import { EthIcon, UsdIcon } from '../common/icons';
 
 const TABLES = 'TABLES';
 const ACTIVITY = 'ACTIVITY';
 
+export const ClaimWinningsSection = () => {
```

Comment:
just wanna put a note saying that I know I left comments below. this component is just a placeholder for the new styles until we get it wired up in an upcoming pr.